cartoframes/core/cartodataframe.py

```diff
@@ -24,16 +46,45 @@ def from_features(cls, features, **kwargs):
         return cls(gdf)
 
     def to_carto(self, *args, **kwargs):
+        """
+        Upload a CartoDataFrame to CARTO.
+        Equivalent to :py:meth:`to_carto <cartoframes.io.to_carto>`.
+
+        Examples:
+
+            .. code::
+                cdf.to_carto(if_exists='replace')
+        """
+
         from ..io.carto import to_carto
         return to_carto(self, *args, **kwargs)
 
     def convert(self, index_column=None, geom_column=None, lnglat_columns=None,
                 drop_index=True, drop_geom=True, drop_lnglat=True):
-        # Magic function
+        """
+        Tries to decode the geometry automatically by looking for coordinates in columns.
+
+        Examples:
+
+            Decode the geometry automatically:
+
+            .. code::
+                cdf = CartoDataFrame(data).convert()
+
+            Passing the geometry column explititely:
+
+            .. code::
+                cdf = CartoDataFrame(data)
+                cdf.convert(geom_column='my_geom_column')
+        """
+
         generate_index(self, index_column, drop_index)
         generate_geometry(self, geom_column, lnglat_columns, drop_geom, drop_lnglat)
         return self
 
     def viz(self, *args, **kwargs):
+        """
+        Create a :py:class:`Map <cartoframes.viz.Map>`. visualization
```

Comment:
Sometimes we use third singular person and other times plural. (I vote for singular :) )